.github/workflows/build-and-test-pr.yaml

```diff
@@ -36,12 +36,21 @@ jobs:
           target-packages: ${{ steps.get-modified-packages.outputs.modified-packages }}
           build-depends-repos: build_depends.repos
 
-      - name: Upload coverage to Codecov
-        if: ${{ steps.get-modified-packages.outputs.modified-packages != '' }}
+      - name: Check the existence of coverage files
+        id: check-file-existence
+        uses: autowarefoundation/autoware-github-actions/check-file-existence@tier4/proposal
+        with:
+          file-path: |
```

Comment:
@KeisukeShima I'm sorry I overlooked it, it was `files`.

https://github.com/autowarefoundation/autoware_common/actions/runs/1781699522

Reply:
Thanks. I'll create a new PR and fix it. But why doesn't this action throw an error? Input argument is required.

Reply:
Mm... it's a mystery. :thinking: